### DISCUSSION TITLE
Unable to set target face for source face

### DISCUSSION THREAD

**Discussion by gocfd** [nextfoam/baram#136] [Q&A]
Hello,

I am unable to successfully run a transient simulation with sliding mesh of a fan. I get an error Unable to set target face for source face 746 with centre: (-0.00139978 0.0283067 -0.0480975). I tried different smaller Time Step Sizes, but nothing seems to help to solve the issue. 
Boundary patches are defined as Interface. 
Case with MRF settings works perfectly fine.

BR

**Comment by kdg5424** [comment]
Hello @gocfd. This is Donggyu from NEXTFOAM.
I think there is a problem with your mesh especially near the interface. The thing that there is no problem with MRF is because MRF doesn't rotate the mesh.
Could you check your mesh near the interface and re-generate mesh again?

Best regards

Donggyu Kim
